vendor/Ooyala/OoyalaVRSDK/skin-config/skin.json

```diff
@@ -150,7 +150,7 @@
     "textEnhancement": "Uniform"
   },
   "upNext": {
-    "showUpNext": true,
```

Comment:
I think we need to create issue for Eric, because we took this config from vendor/OoyalaSDK